src/python/espressomd/shapes.py

```diff
@@ -157,6 +135,7 @@ class Slitpore(ScriptInterfaceHelper):
     pore_mouth : :obj:`float`
     pore_width : :obj:`float`
     upper_smoothing_radius : :obj:`float`
+    dividing_plane : :obj:`float`
```

Comment:
have you added documentation for the new parameter?

Reply:
I have now :-)